client/components/_tests_/header.test.tsx

```diff
@@ -0,0 +1,11 @@
+import Header from "../header";
+import { render } from "@testing-library/react";
+
+describe("Header test", () => {
+  it("is rendering properly header component", () => {
+   const { getByTestId } = render(<Header />);
+   const elementId = getByTestId('header');
```

Comment:
same here

Reply:
i removed all this, i'm getting the links by the container("dom")

Reply:
@eerison i changed a few things after i showed you the new tests, how it is ?